bofire/data_models/strategies/doe.py

```diff
@@ -31,6 +34,22 @@ class DoEStrategy(Strategy):
 
     verbose: bool = False
 
+    objective: OptimalityCriterionEnum = OptimalityCriterionEnum.D_OPTIMALITY
+
+    transform_range: Union[None, Tuple[float, float]] = None
+
+    @field_validator("transform_range")
```

Comment:
Hi @Osburg, I implemented in this PR a `Bounds` type, so that we can reuse it here: https://github.com/experimental-design/bofire/pull/423.

Reply:
nice, thanks, in the latest commit the `Bounds` type is used instead